lib/widgets/gee_build_card.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:appflowy_board/appflowy_board.dart';
+import 'package:flutter/material.dart';
+import 'package:geeruh/theme.dart';
+
+Widget buildCard(AppFlowyGroupItem item) {
+  if (item is RichTextItem) {
+    return _GeeBuildCard(item: item);
+  }
+
+  throw UnimplementedError();
+}
+
+class _GeeBuildCard extends StatefulWidget {
+  final RichTextItem item;
+  const _GeeBuildCard({
+    required this.item,
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<_GeeBuildCard> createState() => _GeeBuildCardState();
+}
+
+class _GeeBuildCardState extends State<_GeeBuildCard> {
+  @override
```

Comment:
Wydaje mi się, że jeżeli nadpisujemy initState i dajemy tam tylko super.initState(), to nie musimy tego robić